### DISCUSSION TITLE
Optimal string alignment

### DISCUSSION THREAD

**Discussion by Shuricks2015** [rapidfuzz/RapidFuzz#402] [Q&A] [answered]
First of all thanks for the amazing work. 

I had a problem today where I got an unexpected result on first view.

```
import rapidfuzz

query = "cases"
str2 = "assessing cabes"
res = rapidfuzz.fuzz.partial_ratio_alignment(s1=query, s2=str2)
print(res)

best_alignment = str2[res.dest_start:res.dest_end]
print(best_alignment)
```

Now the result I get back is that the best alignment is "asses", which I find counterintuitive as you would only need to change a single letter in "cabes" to get to "cases". I searched and found that it is most likely due to the internal use of Indel-distance, as it is similar to Levenshtein with a substitution weight of 2. 

I want to get the optimal alignment using the Levenshtein distance metric. Is there any way that I can achieve that? 
Discussions and help are very welcome.

**Comment by maxbachmann** [comment, marked as answer]
As you noted this is caused by the use of the Indel similarity. There is already https://github.com/rapidfuzz/RapidFuzz/issues/401 which asks for the same feature.
